spark_pipeline_framework/utilities/file_downloader.py

```diff
@@ -84,7 +84,8 @@ def extract_zip_files(
         return None
 
     def download_files_locally(self, filename: str) -> str:
-        prefix = (filename or self.download_to_path or "") + ".."
+        download_to_path = self.download_to_path[7:]
```

Comment:
so this hard-coded value `7:` in the array index - is this always fixed? are there any uses cases where this may break the code for indexing?
I just don't want to have any magic number for an index of the array, because it is error-prone

Reply:
This hard-coded part of the code is only executed when the pipeline is run on the local development system. On S3 we do not access this part of the code, so we will not be facing any indexing issue.
The `7:` is used to remove the protocol from the URI string. (Like `file://` from URI)

Reply:
yep, then let's include the following to make it more explicit, rather than just having the `7:` 
declare a new var `s3_base_path = 'file://'`
get the len of this var, then have this str len into the index value